### PR TITLE
Add original title preference option

### PR DESCRIPTION
**Pull request by robertgzr**
Follow up #4 and add a preference for what language to put into the `OriginalLanguage` field. I also added an option I like personally, which allows to filter which people (especially voice actors) by title preference.

### CODE REVIEW

**Review by h1dden-da3m0n — APPROVED** [review]
Assuming you tested these changes with 10.7.x I give it my LGTM.
And thank you for providing this followup :+1:

**Comment by robertgzr** [comment]
I tested this with `jellyfin/jellyfin:10.7.2`, works great :)

File: Jellyfin.Plugin.AniList/Providers/AniList/ApiModel.cs [hunk 1]
```diff
@@ -191,11 +191,20 @@ public List<string> GetStudioNames()
...
             {
                 foreach (VoiceActor va in edge.voiceActors)
                 {
+                    if (config.FilterPeopleByTitlePreference) {
+                        if (config.TitlePreference == TitlePreferenceType.Japanese && va.language != "JAPANESE") {
+                            continue;
+                        }
+                        if (config.TitlePreference == TitlePreferenceType.Localized && va.language == "JAPANESE") {
+                                continue;
+                        }
```
Comment (h1dden-da3m0n) [inline Jellyfin.Plugin.AniList/Providers/AniList/ApiModel.cs:206]:
```suggestion
                        if (config.TitlePreference == TitlePreferenceType.Localized && va.language == "JAPANESE") {
                            continue;
                        }
```

**Review by h1dden-da3m0n — APPROVED** [review]

**Comment by h1dden-da3m0n** [comment]
hope you don't mind I fixed the one indent (that I likely messed up in my earlier comment/suggestion) and will merge this as soon as CI is green.

**Pull request merged**